holdingsiq.json

```diff
@@ -2226,6 +2218,477 @@
+              "exclusiveMinimum": false,
+              "type": "integer",
+              "format": "int32"
+            }
+          },
+          {
+            "name": "ps",
+            "in": "query",
+            "description": "Page Size",
+            "required": false,
+            "style": "form",
+            "schema": {
+              "maximum": 500,
+              "exclusiveMaximum": false,
+              "minimum": 1,
+              "exclusiveMinimum": false,
+              "type": "integer",
+              "format": "int32"
+            }
+          },
+          {
+            "name": "x-api-key",
+            "in": "header",
+            "description": "Unique code assigned by EBSCO to grant you access to this API.  The API key needs to be a header parameter for every operation in order to access this API. Your production API key will not grant you access to the API through the interactive documentation. You will need a sandbox API key to use the interactive documentation.  If you require a sandbox API key, please contact EBSCO customer support.",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Array of Export objects",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "array",
+                  "items": {
+                    "$ref": "#/components/schemas/Export"
+                  }
+                },
+                "examples": {}
+              }
+            }
+          },
+          "default": {
+            "description": "Unexpected Error",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/errors"
+                }
+              }
+            }
+          }
+        }
+      },
+      "post": {
+        "tags": [
+          "Holdings Resources"
+        ],
+        "summary": "Create Export Request",
+        "description": "This operation enables customers to request an export file of their holdings in one of four formats: \"COST\", \"HoldingsEvent\", \"KBART2\" and \"STANDARD\". Export files will be generated with the most up-to-date holdings information available; however, any changes made to holdings after the request is made may not be reflected in the export. Export files will be deleted after 365 days, and customers are limited to a maximum of 500 export files.\n\n  **Please Note:**  Your production API key to access HoldingsIQ will not give you access to our sandbox. Therefore, your production API key will not give you access to the API through our interactive documentation. You will need a sandbox API key to gain access to the API through our interactive documentation or to use the sandbox directly. If you require a sandbox API key, please contact EBSCO customer support.",
```

Comment:
This operation enables customers to request an export file of their holdings in one of four formats: \"COST\", \"HoldingsEvent\", \"KBART2\" or  \"STANDARD\". Export files will be generated with the most up-to-date holdings information available; however, any changes made to holdings after the request is made may not be reflected in the export. Export files will be deleted after 365 days.  Customers are limited to a maximum of 500 export files.\n\n **Please Note:** Your production API key to access HoldingsIQ will not give you access to our sandbox. Therefore, your production API key will not give you access to the API through our interactive documentation. You will need a sandbox API key to gain access to the API through our interactive documentation or to use the sandbox directly. If you require a sandbox API key, please contact EBSCO customer support.